vitest.config.js

```diff
@@ -0,0 +1,32 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    browser: {
+      provider: 'playwright',
+      enabled: true,
+      headless: true,
+      screenshotFailures: false,
+      instances: [{ browser: 'chromium' }],
+      viewport: { width: 1024, height: 768 },
```

Comment:
There's a bunch of tests that rely on the screen resolution, like the ones around side-by-side mode.

Karma sets bigger screen resolutions, but vitest uses a mobile screen resolution by default. I reckon we probably want to have more control over this and makes sense to set it here.